src/app/src/sensors.json

```diff
@@ -7,7 +7,29 @@
         "Location": "Tinicum",
         "Name": "USGS 01475548 Cobbs Creek at Mt. Moriah Cemetery, Philadelphia",
         "Id": "01475548",
-        "ApiAccess": true
+        "ApiAccess": true,
+        "HealthyRanges": {
```

Comment:
Sorry for making these keys cased this way, the clunkiness has propagated!

Reply:
No worries. The original data had them with a leading uppercase letter. We could reformat these all in the future if it becomes a burden.